compliance_checker/cf/cf.py

```diff
@@ -1893,14 +1894,12 @@ def check_calendar(self, ds):
                 continue
```

Comment:
@benjwadams Could you change the for loop to loop over `cfutil.get_time_variables(ds)` instead?